FirebaseAuthUI/Sources/FUIAuthUtils.m

```diff
@@ -74,4 +76,47 @@ + (nullable UIImage *)imageNamed:(NSString *)name fromBundle:(nullable NSBundle
   }
 }
 
++ (NSString *)randomNonce {
```

Comment:
wdyt about adding a comment saying this method is adapted from the [auth0 docs](https://auth0.com/docs/authorization/mitigate-replay-attacks-when-using-the-implicit-flow#generate-a-cryptographically-random-nonce)? I'm skeptical of my own ability to write any crypto code, so it gives me confidence to say this was ported from a more reputable source.

Reply:
Done.